assignments/function-conversion.js

```diff
@@ -4,22 +4,27 @@
 // console.log("Function was invoked!");
 // };
 // myFunction();
-
+let myFunction = ()=> {console.log("Function was invoked!")}
+myFunction();
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
+let anotherFunction = (param) => {return param;}
```

Comment:
You don't have to write it this way, but keep in back of your mind that if the only thing the function is doing is returning an expression, you'll often see the short implicit return form like this:

```js
let anotherFunction = (param) => param;
```